index.bs

```diff
@@ -6969,9 +6850,9 @@ may return any value, which will be discarded.
     An ECMAScript value |V| is [=converted to an IDL value|converted=]
     to an IDL {{float}} value by running the following algorithm:
 
-    1.  Let |x| be [=ToNumber=](|V|).
-    1.  If |x| is <emu-val>NaN</emu-val>, <emu-val>+Infinity</emu-val> or
-        <emu-val>−Infinity</emu-val>, then <a lt="es throw">throw a <emu-val>TypeError</emu-val></a>.
+    1.  Let |x| be [=?=] [=ToNumber=](|V|).
+    1.  If |x| is <emu-val>NaN</emu-val>, +∞, or −∞,
+         then <a lt="es throw">throw a <emu-val>TypeError</emu-val></a>.
```

Comment:
Fixed in https://github.com/heycam/webidl/commit/36168b74d698015c015d94f2a27414795292217b.